downstream/modules/platform/proc-installing-the-aap-setup-bundle.adoc

```diff
@@ -64,16 +61,16 @@ automationhub_pg_username='automationhub'
 automationhub_pg_password='dbpassword123'
 automationhub_pg_sslmode='prefer'
 ----
-NOTE: The inventory should be kept intact after installation since it is used for backup, restore, and upgrade functions.  Consider keeping a backup copy in a secure location, given that the inventory file contains passwords.
+NOTE: The inventory should be kept intact after installation because it is used for backup, restore, and upgrade functions. Consider keeping a backup copy in a secure location, given that the inventory file contains passwords.
```

Comment:
Just a reminder to correct the format for this admonition in asciidoc. 

```suggestion
[NOTE]
====
The inventory must remain intact after installation because it is used for backup, restore, and upgrade functions. Consider keeping a backup copy in a secure location, given that the inventory file contains passwords.
====
```

Tweaked the wording to use stronger word than "should" (refer to [IBM Word usage for "should"](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#s)